packages/contracts/src/systems/AutoBattleSystem.sol

```diff
@@ -143,6 +209,12 @@ contract AutoBattleSystem is System {
         // clear player if it's defeated, update finishedBoard if else
         if (playerHealth == 0) {
             Utils.clearPlayer(_gameId, _player);
+
+            // //  clear bot
```

Comment:
Since the enemy's health will not decrease, I want to set the goal to be the max round the player can hit. Clearing the bot is for the next time the player enters the game, the bot board is empty. It looks like this part of the code can be moved to the initialized position.